src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -76,7 +77,14 @@ public RsFlash(final String msg, final Level level) {
     public RsFlash(final String msg, final Level level, final String cookie) {
         super(
             new RsWithCookie(
-                cookie, msg,
+                cookie,
+                DatatypeConverter.printBase64Binary(
+                    new StringBuilder(level.getName())
+                        .append('/')
+                        .append(msg)
+                        .toString()
+                        .getBytes(Charset.defaultCharset())
```

Comment:
@dmzaytsev isn't defaultcharset platform dependent? is fo, we might end up with different encodings depending on the platform. I suggest we fix this as utf-8.